pkg/es/config/config.go

```diff
@@ -70,34 +81,56 @@ func (p IndexPrefix) Apply(indexName string) string {
 
 // Configuration describes the configuration properties needed to connect to an ElasticSearch cluster
 type Configuration struct {
```

Comment:
the ordering of the fields feels very random. Even if we are not explicitly nesting them into sub-structs, we should define them in the order so that related fields are next to each other. Can you suggest a grouping?

Reply:
@yurishkuro I made the following 4 groups:

1. connection related configs 
2. es client related configs
3. index related configs 
4. jaeger specific configs 

what do you think?

Reply:
+1.  I would add separator / header comments, like

```
// ---- connection related configs ----
```